DigitalLearningSolutions.Web/ViewModels/MyAccount/MyAccountViewModel.cs

```diff
@@ -23,8 +24,9 @@ DlsSubApplication dlsSubApplication
             ProfilePicture = adminUser?.ProfileImage ?? delegateUser?.ProfileImage;
             Centre = adminUser?.CentreName ?? delegateUser?.CentreName;
             DelegateNumber = delegateUser?.CandidateNumber;
-            AliasId = delegateUser?.AliasId;
             JobGroup = delegateUser?.JobGroupName;
+            CentreEmail = delegateUser?.EmailAddress;
+            DateRegistered = delegateUser?.DateRegistered?.ToString("dd-MM-yyyy");

```

Comment:
Is this the correct date format? We ususally use DigitalLearningSolutions.Data.Helpers.DateHelper.StandardDateFormat